src/sentry/integrations/msteams/webhook.py

```diff
@@ -31,14 +33,44 @@
     CLOCK_SKEW,
 )
 from .link_identity import build_linking_url
-from .utils import build_welcome_card, build_linking_card
+from .utils import build_welcome_card, build_linking_card, ACTION_TYPE
 
 logger = logging.getLogger("sentry.integrations.msteams.webhooks")
 
 # 24 hours to finish installation
 INSTALL_EXPIRATION_TIME = 60 * 60 * 24
 
 
+class MsTeamsIntegrationAssign(analytics.Event):
+    type = "integrations.msteams.assign"
+
+    attributes = (analytics.Attribute("actor_id"), analytics.Attribute("organization_id"))
```

Comment:
@KKaempen You could probably save some lines here by making a base class for this that sets `attributes` so you don't have to repeat the same line four times.